src/types/onyx/ReportNextStep.ts

```diff
@@ -0,0 +1,50 @@
+import {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+
+type Message = {
+    text: string;
+    type: string;
+    action: string;
+};
+
+type Button = {
+    text?: string;
+    tooltip?: string;
+    disabled?: boolean;
+    hidden?: boolean;
+    data?: unknown[];
+};
+
+type ReportNextStep = {
+    /** The message parts of the next step */
+    message: Message[];
+
+    /** The title for the next step */
+    title?: string;
+
+    /** Whether the user should take some sort of action in order to unblock the report */
+    requiresUserAction?: boolean;
+
+    /** The type of next step */
+    type: ValueOf<typeof CONST.NEXT_STEP.type>;
+
+    /** If the "Undo submit" button should be visible */
+    showUndoSubmit?: boolean;
+
+    /** Deprecated - If the next step should be displayed on mobile, related to OldApp */
+    showForMobile?: boolean;
+
+    /** If the next step should be displayed at the expense level */
+    showForExpense?: boolean;
+
+    /** An optional alternate message to display on expenses instead of what is provided in the "message" field */
+    expenseMessage: Message[];
```

Comment:
```suggestion
    expenseMessage?: Message[];
```

According to the propTypes file.